app/assets/javascripts/models/data/account.js.coffee

```diff
@@ -2,82 +2,20 @@ ns = @edsc.models.data
 
 ns.Account = do (ko, ajax=@edsc.util.xhr.ajax) ->
 
-  class Address
-    constructor: ->
-      @street1 = ko.observable("")
-      @street2 = ko.observable("")
-      @street3 = ko.observable("")
-      @city = ko.observable("")
-      @state = ko.observable("")
-      @zip = ko.observable("")
-      @country = ko.observable("")
-      @countryError = ko.observable(false)
-
-    to_json: =>
-      address =
-        street1: @street1()
-        street2: @street2()
-        street3: @street3()
-        city: @city()
-        state: @state()
-        zip: @zip()
-        country: @country()
-
-    from_json: (json) =>
-      @street1(json.street1)
-      @street2(json.street2)
-      @street3(json.street3)
-      @city(json.city)
-      @state(json.state)
-      @zip(json.zip)
-      @country(json.country)
-
-    clear: =>
-      @street1("")
-      @street2("")
-      @street3("")
-      @city("")
-      @state("")
-      @zip("")
-      @country("")
-
-  class Phone
-    constructor: (type) ->
-      @id = ko.observable("")
-      @number = ko.observable("")
-      @type = type
-
-    to_json: =>
-      phone =
-        id: @id()
-        number: @number()
-        phone_number_type: @type
-
   class Account
     constructor: ->
-      @email = ko.observable("")
-      @emailError = ko.observable(false)
-      @firstName = ko.observable("")
-      @firstNameError = ko.observable(false)
-      @lastName = ko.observable("")
-      @lastNameError = ko.observable(false)
-      @organizationName = ko.observable("")
-      @organizationNameError = ko.observable(false)
-      @domain = ko.observable("")
-      @domainError = ko.observable(false)
-      @userType = ko.observable("")
-      @userTypeError = ko.observable(false)
-      @primaryStudyArea = ko.observable("")
-      @primaryStudyAreaError = ko.observable(false)
-      @address = new Address()
-      @region = ko.computed =>
-        if @address.country() == "United States" then "USA" else "INTERNATIONAL"
-
-      @phone = new Phone("BUSINESS")
-      @fax = new Phone("BUSINESS_FAX")
+      @email = ko.observable("Loading ...")
```

Comment:
Initialize the text on contact-info page with loading... until the results come back from URS. Not sure if this is the right way.